### ISSUE TITLE
getPropertyValue not working for IE11

### ISSUE THREAD

**Issue by HandsomeMedia** [postcss/postcss-custom-properties#168]
Trying to access custom properties set in :root does not seem to work on IE11:
`getComputedStyle(document.documentElement).getPropertyValue('--my-var')`
returns empty string despite confirming `--my-var` is properly set.  

Works as expected on Edge and all modern browsers.

**Comment by BrentWMiller** [comment]
@HandsomeMedia IE11 doesn't support CSS variables.

**Comment by HandsomeMedia** [comment]
@BrentWMiller yes, but isn't that the main point of PostCSS custom properties - to polyfill for browsers that don't support it?

**Comment by qkevinto** [comment]
@HandsomeMedia, PostCSS Custom Properties does not polyfill anything but rather transforms your CSS to add in a fallback declaration for the usage of CSS custom properties during build time. PostCSS itself is a CSS transformation/postprocessing tool that's used during build time and has no effect on the client's browser.

**Comment by HandsomeMedia** [comment]
Okay, let me rephrase then: Shouldn't PostCSS transform the :root CSS to add a fallback declaration in IE11, so that we can access it via `getComputedStyle(document.documentElement).getPropertyValue('--my-var')`?

**Comment by coolemur** [comment]
I have the same issue.

I'm thinking of giving https://www.npmjs.com/package/css-vars-ponyfill a go but not sure if this will work as expected in combination with custom properties.

More ideas are welcome. Thank you.

**Comment by coolemur** [comment]
Just tested ponyfill. Didn't work on IE11 (doesn't set/get css variables from js).

**Comment by spiess-demos** [comment]
@coolemur the css-vars-ponyfill module provides javascript access to all custom properties in a callback: https://github.com/jhildenbiddle/css-vars-ponyfill/issues/85#issuecomment-521771857
